probfit/__init__.py

```diff
@@ -9,7 +9,9 @@
  SimultaneousFit
 from .pdf import doublegaussian, ugaussian, gaussian, crystalball, \
     argus, cruijff, linear, poly2, poly3, novosibirsk, \
-    Polynomial, HistogramPdf, cauchy, rtv_breitwigner, johnsonSU
+    Polynomial, HistogramPdf, cauchy, rtv_breitwigner, johnsonSU, \
+    exponential, Exponential
```

Comment:
Do you really need to import `Exponential`?

Reply:
To do like the cruijf distribution
<img width="526" alt="capture d ecran 2018-11-14 a 11 39 10" src="https://user-images.githubusercontent.com/24250309/48477457-ed617a80-e801-11e8-80b0-b6cc0742a4c0.png">